tests/redis-trib.rb

```diff
@@ -0,0 +1,1402 @@
+#!/usr/bin/env ruby
```

Comment:
Yes that is what i wanted. It makes it alot more clear what you acctually changed to make it work.